src/integrationTest/java/uk/gov/hmcts/reform/wataskmanagementapi/controllers/GetTaskByIdControllerTest.java

```diff
@@ -603,7 +604,7 @@ void should_return_404_when_initiation_request_failed_to_retrieve_data_from_ccd(
                 jsonPath("$.title").value("Generic Server Error"),
                 jsonPath("$.status").value(500),
                 jsonPath("$.detail").value("Generic Server Error: The action could not be "
-                                           + "completed because there was a problem when initiating the task.")
+                                               + "completed because there was a problem when initiating the task.")
```

Comment:
Probably still need to discuss this as not sure it's correct still.  I notice you've probably just reverted here so it may be better to putting the fix in another Jira but I'd like to understand what's going on first.

Reply:
discussed and happy that it's working as expected.